src/components/button/button_group/button_group_button.tsx

```diff
@@ -129,6 +129,7 @@ export const EuiButtonGroupButton: FunctionComponent<Props> = ({
         'data-text': innerText,
         title: innerText,
       }}
+      title={isIconOnly ? label?.toString() : undefined}
```

Comment:
🤦‍♀️  It looks like we had already setup a `title` property but was sending it to the wrong component (via `textProps` instead of directly on the `EuiButtonDisplay`). I'm wondering if we can just move the exiting `title` prop this new one. I think it's also a better way to capture the correct label text in case the label passed in was from a component like `<span>Label</span>`.

```suggestion
      }}
      title: innerText,
```

Reply:
I also think adding it for all (not just icon only) versions will help in case the text label is truncated.

Reply:
good idea, ill add in a commit making those changes